docs/upgrading-pega-deployment-zero-downtime.md

```diff
@@ -0,0 +1,179 @@
+# Upgrading Pega Platform in your deployment with zero-downtime
+
+After you deploy Pega Platform™ on your kubernetes environment, you can use a Pega-provided Docker image-based process to upgrade your Pega software with zero-downtime. The following procedures are written for any level of user, from a system administrator to a development engineer who wants to use helm charts and Pega Docker images to upgrade the Pega software you have deployed in any supported kubernetes environment.
+
+For the latest Pega Community articles about Pega-provided docker images, see [About client managed cloud](https://community.pega.com/knowledgebase/articles/client-managed-cloud/86/about-client-managed-cloud).
+
+## Kubernetes-based zero-downtime upgrade process overview
+
+Pega Infinity™ upgrades starting from version 8.4.2 feature a near-zero-downtime process, so your environment remains available you can continue working in your application during the upgrade. You complete a zero-downtime upgrade using Pega helm charts, which automate the Pega zero-downtime upgrade process using the four schemas used in the version 8.4 and later Pega patch and upgrade processes:
+
+- Data schema – your existing data schema. This will be your data schema after the zero-downtime upgrade.
+- Rules schema – your existing rules schema. This schema is upgraded during the zero-downtime upgrade.
+- A temporary data schema (Dtmp) – a schema you create on the same database to host some required data-specific tables. You discard this schema after the upgrade.
+- New rules schema (Rn) – a rules schema you create on the same database for staging the rules during the upgrade. This schema will become the rules schema after the upgrade.
+
+Before beginning your upgrade, you must create empty schemas - the new rules schema and new temporary data schema  - in your existing database as shown in the image below.
```

Comment:
This step is automated now, just need to specify the schema names. Creation happens automatically. @arvasrikanth